inst/app/global.R

```diff
@@ -9,7 +9,18 @@
 # Packages ----------------------------------------------------------------
 
 ## data wrangling
+library(dplyr)
+library(tidyr)
+library(hkdatasets)
 
 ## shiny-related
 library(shiny)
 library(shinydashboard)
+library(ggplot2)
+library(plotly)
```

Comment:
Could you `renv::snapshot()` these packages, please?

Reply:
Yes! Just did so.